package.json

```
@@ -26,10 +26,10 @@
     "react-router-dom": "7.18.1"
   },
   "devDependencies": {
-    "@babel/core": "8.0.1",
-    "@babel/plugin-transform-react-jsx-development": "8.0.1",
-    "@babel/preset-react": "8.0.1",
-    "@babel/types": "8.0.4",
+    "@babel/core": "7.29.7",
```

Biome (2.5.3)

[error] The key `@babel/core` was already declared.

(lint/suspicious/noDuplicateObjectKeys)